app/controllers/users_controller.rb

```diff
@@ -128,7 +129,8 @@ def new
   def request_new
     flash[:warn] = "If you are a student, please contact your teaching staff to get your Expertiza ID."
     @user = User.new
-    @rolename = Role.find_by(name: params[:role])
+    @role = Role.new
```

Comment:
Any reason for creating a new empty `Role`?

Reply:
Do you suggest that we should remove this?

Reply:
We have removed this